projects/tiempoCarlaSergio/src/components/Cabecera.vue

```diff
@@ -10,7 +10,8 @@
   </div>-->
   <div>
     <b-navbar>
-      <b-navbar-brand href="#">El tiempo</b-navbar-brand>
+      <img src="../assets/logo.png" alt="logo" class="logo">
+      <!--<b-navbar-brand href="#">El tiempo</b-navbar-brand>-->
```

Comment:
Tal vez os coloque ya el logo en su sitio si utilizáis b-navbar-brand como tenéis comentado.

```
    <b-navbar-brand href="#">
        <img src="../assets/logo.png" alt="logo" class="logo">
    </b-navbar-brand>
```

También en lugar de href="#" lo suyo es que fuera un link a la pantalla "home"

Reply:
Evitar que sea relativo, mejor utilizar @/assets...